storage-resize-images/functions/src/index.ts

```diff
@@ -228,20 +273,26 @@ const resizeImage = async ({
       metadata.metadata.firebaseStorageDownloadTokens = uuid();
     }
 
-    // Generate a resized image using Sharp.
-    logs.imageResizing(resizedFile, size);
+    // Generate a resized image buffer using Sharp.
+    logs.imageResizing(modifiedFile, size);
+    const resizedImageBuffer = await resize(originalFile, size);
+    logs.imageResized(modifiedFile);
 
-    await resize(originalFile, resizedFile, size);
+    // Generate a converted image type buffer using Sharp.
+    logs.imageConverting(fileExtension, config.imageType);
+    const convertedFormatImageBuffer = await convertType(resizedImageBuffer);
+    logs.imageConverted(config.imageType);
```

Comment:
If user hasn't specified image type conversion, this logs out `Converted image to undefined`